payloadgenerator/associated_fields.py

```diff
@@ -0,0 +1,33 @@
+from base64 import b64encode;
```

Comment:
It would be great if it had a description describing what the generator does to the payloads.